data/netplan-networkd-wait-online@.service

```diff
@@ -0,0 +1,26 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
note: This should be compatible with our GPL-3 code, according to https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility